src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -684,11 +685,15 @@ void Load_IC_Prof_CCSN()
 void Record_CCSN()
 {
 
-// (1) record quantities at the center
+// (1) shock detection
+   if ( CCSN_Prob != Migration_Test  &&  CCSN_Is_PostBounce )
+      Detect_Shock();
+
```

Comment:
Maybe we need to add one more empty line here?

Reply:
Added